lib/teleterm/services/connectmycomputer/connectmycomputer.go

```diff
@@ -235,7 +235,7 @@ func (s *RoleSetup) syncResourceUpdate(ctx context.Context, accessAndIdentity Ac
 
 	err = updateFunc(ctx)
 	if err != nil {
-		return trace.Wrap(err, "calling update function")
+		return trace.Wrap(err)
```

Comment:
I removed this custom message here, because if an error got triggered, the final error would look like this:

```
User Message: calling update function
	updating role connect-my-computer-rav
		resource revision does not match, it may have been concurrently created|modified|deleted; please work from the latest state, or use --force to overwrite] apiserver/middleware.go:40
```